stacks/pipelines_stack.py

```diff
@@ -63,16 +63,21 @@ def __init__(self, scope: core.Construct, construct_id: str, **kwargs) -> None:
                                                                        "dateStamp=$(date +%Y%m%d)",
                                                                        "echo Repo Name: $repoName",
                                                                        "echo Commit ID: $commitId",
-                                                                       "ls",
                                                                        "existingTags=$(aws ecr list-images --repository-name $repoName --output json --query 'imageIds[*].imageTag')",
-                                                                       "echo Tags that already exist in this ECR repo: $existingTags"
+                                                                       "echo Tags that already exist in this ECR repo: $existingTags",
 
 
                                                                        # TODO: Unit tests
                                                                        # TODO: Static Analysis (tool TBD)
 
+                                                                       """
+                                                                       if test -f get-artifacts.sh; then
+                                                                         echo 'get-artifacts.sh file present'
+                                                                         ./get-artifacts.sh
+                                                                       fi
+                                                                       """,
```

Comment:
@JonHolman looks good for me. Could you arrange a small meeting with Yasser, Fabrice, Dheeraj and me so that we all could be on the same page on using `get-artifacts.sh` script